frontend/src/views/BlogHomePage.vue

```diff
@@ -7,47 +7,49 @@
     <img class="hidden md:block absolute bottom-0 right-20" src="/static/images/plants.png" />
   </div>
   <div class="fr-container my-6">
-    <div class="flex justify-center" v-if="loading">
+    <div v-if="isFetching" class="flex justify-center">
       <ProgressSpinner />
     </div>
-    <p v-else-if="blogPostCount === 0">Nous n'avons pas encore d'articles de blog</p>
-    <div v-else>
-      <div class="grid grid-cols-12 gap-4">
-        <BlogCard
-          class="col-span-12 sm:col-span-6 md:col-span-4"
-          v-for="post in visibleBlogPosts"
-          :key="post.id"
-          :post="post"
+    <template v-if="blogPosts">
+      <div v-if="blogPosts.results.length > 0">
+        <div class="grid grid-cols-12 gap-4">
+          <BlogCard
+            class="col-span-12 sm:col-span-6 md:col-span-4"
+            v-for="post in blogPosts?.results"
```

Comment:
```suggestion
            v-for="post in blogPosts.results"
```
Ici on est sur de les avoir non ? Vu qu'on est à l'intérieur d'un `v-if="blogPosts.results.length > 0"`

Reply:
Tout à fait, merci !